pages/common/brotli.md

```diff
@@ -5,20 +5,20 @@
 
 - Compress a file, creating a compressed version next to the file:
 
-`brotli {{file.ext}}`
+`brotli {{path/to/file.ext}}`
 
 - Decompress a file, creating an uncompressed version next to the file:
 
-`brotli -d {{file.ext}}.br`
+`brotli -d {{path/to/file.ext}}.br`
 
 - Compress a file specifying the output filename:
 
-`brotli {{file.ext}} -o {{compressed_file.ext.br}}`
+`brotli {{path/to/file.ext}} -o {{compressed_file.ext.br}}`
 
 - Decompress a Brotli file specifying the output filename:
 
-`brotli -d {{compressed_file.ext.br}} -o {{file.ext}}`
+`brotli -d {{compressed_file.ext.br}} -o {{path/to/file.ext}}`
```

Comment:
```suggestion
`brotli -d {{path/to/compressed_file.ext.br}} -o {{path/to/output_file.ext}}`
```

Reply:
Script should handle all placeholders which contain 'file' in their name, and not slashes I guess to make it work. Fixing all pages manually will be a huge pain. :)

Reply:
Fixed via script already. :)